### PR TITLE
fix(mobile): update app launch time docs

### PR DESCRIPTION
**Pull request by ghost**
Updated app launch time docs so that Android and iOS pages are more similar (in response to CF ticket NR-276587)

### CODE REVIEW

File: src/content/docs/mobile-monitoring/new-relic-mobile-android/install-configure/configure-app-launch-time-android-apps.mdx [hunk 1]
```diff
@@ -5,37 +5,41 @@ tags:
...
-
-Here's a sample query for capturing cold and hot launch times: 
-
-```sql
-  SELECT average(newrelic.timeslice.value ) AS 'AppLaunch/Cold' FROM Metric WHERE metricTimesliceName = 'AppLaunch/Cold' AND entity.guid = 'YOUR_APP_TOKEN' SINCE 1664218800000 UNTIL 1664220600000 TIMESERIES
+With the Android agent, you can report app launch time, also known as "cold time" and "hot time." 
+
+## Configure app launch time [#configure-app-launch-time]
+
+To enable the Android agent to report app launch time, addd the following to your application's `AndroidManifest.xml`:
```
Comment (ally-sassman) [inline src/content/docs/mobile-monitoring/new-relic-mobile-android/install-configure/configure-app-launch-time-android-apps.mdx:19]:
```suggestion
To enable the Android agent to report app launch time, add the following to your application's `AndroidManifest.xml`:
```

File: src/content/docs/mobile-monitoring/new-relic-mobile-android/install-configure/configure-app-launch-time-android-apps.mdx [hunk 2]
```diff
@@ -5,37 +5,41 @@ tags:
...
+
+```xml
+<application>
+
+    <!-- make sure to provide unique authorities if apply to multiple apps on the same device -->
```
Comment (homelessbirds) [inline src/content/docs/mobile-monitoring/new-relic-mobile-android/install-configure/configure-app-launch-time-android-apps.mdx:24]: For this line "if apply to multiple apps on the same device" should it be "if it applies?" or "if applying?"

**Comment by homelessbirds** [comment]
Just left one comment (in addition to your committed suggestion of addd-->add) but otherwise looks good to me!

**Review by homelessbirds — DISMISSED** [review]

File: src/content/docs/mobile-monitoring/new-relic-mobile-android/install-configure/configure-app-launch-time-android-apps.mdx (hunk 2 above)
Comment (ally-sassman) [inline src/content/docs/mobile-monitoring/new-relic-mobile-android/install-configure/configure-app-launch-time-android-apps.mdx:24]:
```suggestion
    <!-- Make sure to provide unique authorities if applied to multiple apps on the same device -->
```

**Pull request merged**